minecraft/1.7/src/main/java/nova/core/wrapper/mc/forge/v17/wrapper/block/BlockConverter.java

```diff
@@ -136,7 +138,8 @@ private void registerNovaBlock(BlockFactory blockFactory) {
 		FWBlock blockWrapper = new FWBlock(blockFactory);
 		blockFactoryMap.put(blockFactory, blockWrapper);
 		NovaMinecraft.proxy.registerBlock(blockWrapper);
-		GameRegistry.registerBlock(blockWrapper, FWItemBlock.class, blockFactory.getID());
+//		GameRegistry.registerBlock(blockWrapper, FWItemBlock.class, blockFactory.getID());
```

Comment:
Can't you inject mods into FML, then when they get called by FML you register stuff with the correct IDs?